extra/reset-password.js

```diff
@@ -27,8 +27,17 @@ const main = async () => {
             console.log("Found user: " + user.username);
 
             while (true) {
-                let password = await question("New Password: ");
-                let confirmPassword = await question("Confirm New Password: ");
+                let password;
+                let confirmPassword;
+
+                # When called with "--new_password" argument for unattended modification (e.g. npm run reset-password -- --new_password=secret) 
```

Comment:
Please test the contribution before creating a PR.

This won't run, as JS usess backslashes

See 
![image](https://github.com/louislam/uptime-kuma/assets/26258709/1af7b052-fbd5-4485-ac82-72f4b955286f)


```suggestion
                // When called with "--new_password" argument for unattended modification (e.g. npm run reset-password -- --new_password=secret) 
```

Reply:
You are right, I admit I added this comment after testing it (and I'm more a Bash coder :P).

I'll fix this and run the formatter/linter.

Reply:
Comment now uses correct `//`.